src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -146,21 +147,30 @@ public void updatePerson(ReadOnlyPerson target, ReadOnlyPerson editedPerson)
 
     //@@author freesoup
     @Override
-    public void removeTag(Tag tag) throws PersonNotFoundException, DuplicatePersonException {
+    public void removeTag(Tag tag) throws PersonNotFoundException, DuplicatePersonException,
```

Comment:
Not too sure how `PersonNotFoundException` and `DuplicatePersonException` is thrown when trying to remove a tag from everybody in the address book.

Reply:
Yes, I realised this as well. I have added a assert statement to assert that these shouldnt be thrown